app/model/models.js

```diff
@@ -232,5 +233,29 @@ module.exports = {
                 " WHERE `models`.`model_id` ="+dbpool.escape(m)+";";
 
         queryHandler(q, callback);
-    }
+    },
+
+    JOB_SCHEMA : joi.object().keys({
+        ENV_JOB_ID: joi.string()
+    }),
+
+    createRFM: function(data, callback){
+        const payload = JSON.stringify(
+            {
+                ENV_JOB_ID: `${data.jobId}`
+            }
+        )
+        return q.ninvoke(joi, 'validate', payload, this.JOB_SCHEMA)
+            .then(async () => {
+                data.kubernetesJobName = `arbimon-rfm-${new Date().getTime()}`;
+                const jobParam = jsonTemplates.getRfmTemplate('arbimon-rfm', 'job', {
+                    kubernetesJobName: data.kubernetesJobName,
+                    imagePath: k8sConfig.soundscapeImagePath,
```

Comment:
this is the problem, you have to use `arbimon-rfm`

Reply:
fixed